docs/roadmap-2021.md

```diff
@@ -0,0 +1,184 @@
+
+**Theme:** Multi-Tenant environment support
+
+**GitHub Issue:** TBD
+
+**Owner:** Rick Rineholt
+
+### Autotrigger Sugar Controller
+[Description TBD]
+
+**GitHub Issue:** https://github.com/knative/eventing/issues/4547
+
+**Owner:** Scott Nichols
+
+### Remove v1beta1 API [eventing, flows, messaging]
+[Description TBD]
+
+**GitHub Issue:** https://github.com/knative/eventing/issues/4836
+
+**Owner:** Ville Aikas
+
+## Next 6 Months (Midterm)
+### Protocol negotiation contract
+[Description TBD]
+
+**Theme:** Multiple protocol and protocol option support
+
+**GitHub Issue:** TBD
+
+**Owner:** Grant Rodgers
+
+### Preflight protocol negotiation
+[Description TBD]
+
+**Theme:** Multiple protocol and protocol option support
+
+**GitHub Issue:** TBD
+
+**Owner:** Grant Rodgers
+
+## Icebox (Wishlist)
+### Broker Performance Tests
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+**Owner:** TBD 
+
+### Cross Namespace Eventing
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+**Owner:** Grant Rodgers
+
+### Advanced Workflow and Streaming processing
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+**Owner:** ?
+
+## Won’t Do
+__These are the issues the working group decided to not work on whether for being out of the WG scope or some other reason
+
+# Eventing Sources
+## Next 3 Months (Short-term)
+### Introduce and define Multi-tenant Sources
+[Description TBD]
+
+**Theme:** Multi-Tenant environment support
+
+**GitHub Issue:** TBD
+
+**Owner:** Scott Nichols
+
+### Multi-Tenant Kafka Source
+[Description TBD]
+
+**Theme:** Multi-Tenant environment support
+
+**GitHub Issue:** https://github.com/knative-sandbox/eventing-kafka/issues/219
+
+**Owner:** Lionel Villard
+
+### GitHub Vanity domain support
+[Description TBD]
+
+**Theme:** Multi-Tenant environment support
+
+**GitHub Issue:** https://github.com/knative-sandbox/eventing-github/issues/65
+
+**Owner:** Lionel Villard
+
+### Find Maintainers for source repos
+[Description TBD]
+
+**Theme:** No repo left behind
+
+**GitHub Issue:** We should create one if we think this is valuable
+
+**Owner:** Ville Aikas
+
+
+### v1 for Core Sources
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+**Owner:** ?
+
+## Next 6 Months (Midterm)
+### Multi-Tenant RedisStream Source
+[Description TBD]
+
+**Theme:** Multi-Tenant environment support
+
+**GitHub Issue:** TBD
+
+**Owner:** Lionel Villard
+
+## Icebox
+### Removal of v1alpha1 Sources
+[Description TBD]
+
+**GitHub Issue:** TBD
+
+## Won’t Do
+__These are the issues the working group decided to not work on whether for being out of the WG scope or some other reason
+
+# Event Delivery
+## Next 3 Months (Short-term)
+### Kafka Code Share Improvements
+[Description TBD]
+
+**Theme:** Kafka Code Share Improvements
+
+**GitHub Issue:** TBD
+
+**Owner:** Matthias
```

Comment:
Ali, Eric, Matthias , Travis

Reply:
@travis-minke-sap @aliok

Reply:
👍 Yeah, we are focused on our internal delivery deadlines for the short term.  There are still a few changes to the distributed kafkachannel that I'd like to make to move us toward commonality but nothing shared.  Thanks Matthias!

Reply:
@matzew we agreed in the roadmap call that we would like to keep the "Owner" a single person. An item can have one owner and many people driving the work. He's just the single point of contact on the roadmap doc.